processors/games/HungerGames.js

```diff
@@ -1,8 +1,221 @@
+/* eslint-disable camelcase */
 /*
-* Blitz Survival Games
+ * Blitz Survival Games
  */
+
+const {
+  getRatio,
+  getWeeklyStat,
+  getMonthlyStat,
+} = require('../../util/utility');
+
 module.exports = ({
-  coins = 0,
-}) => ({
+  // General
   coins,
-});
+  deaths,
+  kills,
+  wins,
+  wins_teams,
+  damage_taken,
+  taunt_kills,
+  potions_drunk,
+  damage,
+  mobs_spawned,
+  time_played,
+  arrows_hit,
+  games_played,
+  potions_thrown,
+  arrows_fired,
+  blitz_uses,
+
+  monthly_kills_a,
+  monthly_kills_b,
+  weekly_kills_a,
+  weekly_kills_b,
+
+  // Kits
+  arachnologist,
+  archer,
+  armorer,
+  astronaut,
+  baker,
+  blaze,
+  creepertamer,
+  diver,
+  farmer,
+  fisherman,
+  florist,
+  golem,
+  guardian,
+  horsetamer,
+  hunter,
+  'hype train': hype_train,
+  jockey,
+  knight,
+  meatmaster,
+  necromancer,
+  paladin,
+  pigman,
+  reaper,
+  reddragon,
+  rogue,
+  scout,
+  'shadow knight': shadow_knight,
+  slimeyslime,
+  snowman,
+  speleologist,
+  tim,
+  toxicologist,
+  troll,
+  viking,
+  warlock,
+  wolftamer,
+
+  // Equipped
+  aura,
+  chosen_taunt: taunt,
+  chosen_victorydance: victory_dance,
+  chosen_finisher: finisher,
+
+  // Misc
+  packages,
+  defaultkit,
+  rambo_wins,
+  combatTracker,
+  autoarmor,
+  random_wins,
+  togglekillcounter,
+
+  // Other
+  ...rest
+}) => {
+  // Gets all stats whose key started with prefix
+  const getKitStat = (prefix, filter) => {
+    let entries = Object.entries(rest).filter(([key]) => key.startsWith(prefix));
+
+    if (filter) {
+      entries = entries.filter(([key]) => filter(key));
+    }
+
+    const stats = entries.reduce((prev, [key, val]) => {
+      const normalizedKey = key.replace(prefix, '').replace(' ', '_');
+      return {
+        ...prev,
+        [normalizedKey]: val,
+      };
+    }, {});
+
+    // Return undefined instead of empty object
+    return Object.keys(stats).length === 0 ? undefined : stats;
+  };
+
+  const inventories = Object.entries(rest)
+    .filter(([key]) => key.endsWith('Inventory'))
+    .filter(v => typeof v[1] === 'object')
+    .map(v => [v[0].replace('Inventory', '').replace(' ', '_').toLowerCase(), v[1]])
+    .reduce((prev, curr) => {
+      const key = curr[0];
+      const val = Object.values(curr[1]);
+
+      return {
+        ...prev,
+        [key]: val,
+      };
+    }, {});
+
+  return {
+    coins,
+    deaths,
+    kills,
+    kd: getRatio(kills, deaths),
+    wins,
+    team_wins: wins_teams,
+    win_loss: getRatio(wins + wins_teams, wins + wins_teams + deaths),
+    weekly_kills: getWeeklyStat(weekly_kills_a, weekly_kills_b),
+    monthly_kills: getMonthlyStat(monthly_kills_a, monthly_kills_b),
+    rambo_wins,
+    random_wins,
+    kits_levels: {
```

Comment:
Having values 1-indexed would make the API more accessible for newer developers, and values can be more easily used in ui.

Reply:
Ahh I see, the kit levels start at zero. Right, adding one would make sense